tutoraspects/plugin.py

```diff
@@ -393,6 +394,10 @@
         ("DBT_PROFILE_TARGET_DATABASE", "reporting"),
         ("RUN_ASPECTS_DOCS", False),
         ("DBT_HOST", "dbt.{{LMS_HOST}}"),
+        #####################
+        # MFE Customizations
+        # Aspects can enable plugins to show in-context metrics in the Authoring MFE
+        ("ASPECTS_ENABLE_STUDIO_IN_CONTEXT_METRICS", True),
```

Comment:
I'm not sure if we want this on by default yet, given the authoring MFE version will need to be pretty recent for it to work and otherwise Aspects works back to the Redwood release. Can you add some documentation here about what the specific requirements are for this feature to work? We can roll those up into the Aspects documentation as this feature lands.